src/version.rs

```diff
@@ -68,9 +71,90 @@ pub struct VersionLinks {
     pub dependencies: String,
     pub version_downloads: String,
     pub authors: String,
+    pub build_info: String,
+}
+
+#[derive(Insertable, AsChangeset)]
+#[table_name="build_info"]
+#[primary_key(version_id, rust_version, target)]
+struct NewBuildInfo {
+    version_id: i32,
+    rust_version: String,
+    target: String,
+    passed: bool,
+}
+
+#[derive(Identifiable, Queryable, Associations)]
+#[belongs_to(Version)]
+#[table_name="build_info"]
+#[primary_key(version_id, rust_version, target)]
+pub struct BuildInfo {
+    version_id: i32,
+    rust_version: String,
+    target: String,
+    passed: bool,
+}
+
+pub const BUILD_INFO_FIELDS: (build_info::version_id, build_info::rust_version, build_info::target, build_info::passed) = (build_info::version_id, build_info::rust_version, build_info::target, build_info::passed);
```

Comment:
Do we even need the `created_at` and `updated_at` fields if we aren't selecting them?